piwebapi_samples/Data Analysis/PIUploadUtility/PIUploadUtility/PIUploadUtility/Program.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Xml;
+using System.IO;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace PIUploadUtility
+{
+    class Program
+    {
+        static readonly string defaultConfigFile = @"..\..\..\..\..\test_config.json";
```

Comment:
Why is this so high in folder hierarchy? Is the pipeline dropping the config file in the root of the repo? Usually the pipeline will put the config file inside the project, in this case inside `piwebapi_samples/Data Analysis/PIUploadUtility`

Reply:
The config file is being used by both Jupyter Notebook (Sandbox.ipynb) and `PIUploadUtility` to populate the values of AssetServer and other things. Therefore it needs to be on root. However, I have reduced the folder path one level in my next commit.